src/XIOPAK.H

```diff
@@ -114,6 +114,11 @@ uintmax_t FC dskDrvFreeSpace();
 SI     FC xffilcmp( const char * fileName1, const char * fileName2 );
 SEC	xfclear(XFILE* xf);
 
+void getPathRootName(const char* path, char* rootName);
+void getPathRootDirectory(const char* path, char* rootDirectory);
+void getPathRootStem(const char* path, char* rootStem);
```

Comment:
What is a "root stem"? I imagine you are referring to "[stem](https://en.cppreference.com/w/cpp/filesystem/path/stem)", or the "file base name", but I don't think the word "root" belongs in the name of the function. "file base name" is more intuitive to me than "stem", but filesystem seems to have standardized on "stem". I could go either way.

Reply:
It's intuitive to me now, but if C++ is trying to standardize on something else, that might be a more compelling direction.